src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -397,8 +397,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             // GH#11315: Always do this, even if they don't have acrylic on.
             _renderEngine->EnableTransparentBackground(_isBackgroundTransparent());
 
-            THROW_IF_FAILED(_renderEngine->Enable());

```

Comment:
this seemed important

Reply:
It wasn't implemented by AtlasEngine! It initializes itself lazily.